src/main/java/io/stargate/sgv2/jsonapi/service/resolver/matcher/FilterMatcher.java

```diff
@@ -123,7 +145,7 @@ protected Capture(Object marker) {
       this.marker = marker;
     }
 
-    public List<FilterOperation<?>> match(ComparisonExpression t) {
+    public List<FilterOperation> match(ComparisonExpression t) {
```

Comment:
Raw use of parameterized class 'FilterOperation'

Reply:
fixed.

Reply:
Discuss